linkerd/app/admin/src/stack.rs

```diff
@@ -85,13 +89,15 @@ impl Config {
         R: FmtMetrics + Clone + Send + Sync + Unpin + 'static,
         B: Bind<ServerConfig>,
         B::Addrs: svc::Param<Remote<ClientAddr>> + svc::Param<Local<ServerAddr>>,
+        P: inbound::policy::GetPolicy + Send + Sync + 'static,
+        P::Future: Send,
     {
         let (listen_addr, listen) = bind.bind(&self.server)?;
 
+        let (ready, latch) = crate::server::Readiness::new();
```

Comment:
Did this need to move? It seemed a little clearer where it was.

Reply:
i think this moved because at one point, i had changed the function to spawn rather than being `async fn` and half of the readiness latch needed to be outside the async block. it can move back now since that's no longer needed.